CMakeLists.txt

```diff
@@ -53,7 +53,8 @@ if (EVSE_SECURITY_INSTALL)
     )
 endif()
 
-if(BUILD_TESTING_EVSE_SECURITY)
+if((CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME OR ${PROJECT_NAME}_BUILD_TESTING) AND BUILD_TESTING)
```

Comment:
```suggestion
if( (CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME AND BUILD_TESTING) OR ${PROJECT_NAME}_BUILD_TESTING)
```

Reply:
I am following the pattern in the Modern CMake book: https://cliutils.gitlab.io/modern-cmake/chapters/testing.html